solr/core/src/test/org/apache/solr/cloud/LeaderVoteWaitTimeoutTest.java

```diff
@@ -179,74 +179,78 @@ public void basicTest() throws Exception {
 
   @Test
   public void testMostInSyncReplicasCanWinElection() throws Exception {
-    final String collectionName = "collection1";
-    CollectionAdminRequest.createCollection(collectionName, 1, 3)
+    // final String collectionName = "collection1";
```

Comment:
I think this could be 

```
final String collectionName = DEFAULT_TEST_COLLECTION_NAME;
```

which would make the diff a lot smaller?

Reply:
I think the value of this PR is to use the same pattern everywhere (even though maybe the diff is bigger).    I want to only have a variable called collectionName if that is somehow important to the function of the test..  when I see the variable I immediately think, oh, it might change, and I need to think about that as I read the test.    Where as seeing the constant everywhere means I know that the name of the collection isn't important to the function of the test...   So thats why I made these changes, and yeah, it did make the diff bigger...   Does that argument sway your opinion?   I could split up this patch into multiple patches too if that was better...   My end goal on this PR is to use the same constant the same way everywhere that it makes sense......